src/sass/components/_global.scss

```diff
@@ -0,0 +1,5 @@
+html,
+body {
+  overflow-x: hidden;
```

Comment:
Zły pattern - jeśli strona się scrolluje to jest to dla nas informacje, że coś się źle dzieje - nie powinniśy temu zapobiegać